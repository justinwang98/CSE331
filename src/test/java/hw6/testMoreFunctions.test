# Test the parser on a graph that contains several nodes
# with two edges each.

LoadGraph heroes staffSuperheroes.tsv
ListNodes heroes
ListChildren heroes Notkin-of-the-Superhuman-Beard