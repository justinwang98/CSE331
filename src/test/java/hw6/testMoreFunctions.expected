# Test the parser on a graph that contains several nodes
# with two edges each.

loaded graph heroes
heroes contains: Ernst-the-Bicycling-Wizard Grossman-the-Youngest-of-them-all Notkin-of-the-Superhuman-Beard Perkins-the-Magical-Singing-Instructor
the children of Notkin-of-the-Superhuman-Beard in heroes are: Ernst-the-Bicycling-Wizard(CSE331) Ernst-the-Bicycling-Wizard(CSE403) Grossman-the-Youngest-of-them-all(CSE331) Perkins-the-Magical-Singing-Instructor(CSE331)